mono/metadata/object.c

```diff
@@ -3421,15 +3447,13 @@ mono_field_get_addr (MonoObject *obj, MonoVTable *vt, MonoClassField *field)
  * <pre>
  * int i;
  *
- * mono_field_get_value (obj, int_field, &i);
+ * mono_field_get_value_internal (obj, int_field, &i);
  * </pre>
  */
 void
 mono_field_get_value (MonoObject *obj, MonoClassField *field, void *value)
 {
-	MONO_ENTER_GC_UNSAFE;
```

Comment:
Perhaps this PR should not have any GC mode change changes, i.e. do them in a separate PR.